src/args.rs

```diff
@@ -32,6 +32,7 @@ pub const REDIS_KEY: &str = "log-redis-key";
 pub const REDIS_PORT: &str = "log-redis-port";
 pub const ROOT_DIR: &str = "root-dir";
 pub const STORAGE_BACKEND_TYPE: &str = "sbe-type";
+pub const ENABLE_SITEMD5: &str = "enable-sitemd5";
```

Comment:
I wonder if the Linux convention with command line flags without values is not that it always enable the feature? If so then the ‘enable-‘ prefix is redundant